filemover/main.py

```diff
@@ -59,7 +57,8 @@ def move_files(file_folder_map: dict):
     '''
     for folder, files in file_folder_map.items():
         for file in files:
-            move(file, folder)
+            os.makedirs(os.path.join(folder_path, folder), exist_ok=True)
```

Comment:
Since this need to be executed only once for each folder, I think it should be moved out of second for loop. 

Also, I think to increase code readability, the common expression should be initialized as variables, such as:
```
old_file_path = os.path.join(folder_path, file)
new_file_path = os.path.join(folder_path, folder)
```

Reply:
Also, not sure, but I think that this line creates folders where the files will be moved. The same task is being done in `create_folder` function called on line 106. If yes, I think this logic should be moved to the `create_folder` function and folder should be created outside `move_files`. The reason to keep the folder creation and file moving logic separate is to keep the code flow simpler and increase readability.

Reply:
`folder_path` variable name needs to change. like input_dir, clean_dir, or something different.